### PR TITLE
feat(feed-item-row): DP-119844 add slot for displayName

### PR DESCRIPTION
**Pull request by ninamarina**
# Feed item row: add slot for displayName

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
[DP-119844]

## :book: Description
We need a button in place of the displayName to be able to trigger the Hovercard.

image

[DP-119844]: https://dialpad.atlassian.net/browse/DP-119844?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js [hunk 1]
```diff
@@ -102,6 +102,33 @@ export const argTypesData = {
...
+    name: 'displayName',
+    description: 'Slot for display name',
+    control: 'text',
+    table: {
+      category: 'slots',
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
```
Comment (braddialpad) [inline packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js:115]: This doesn't work in storybook when you make the prop blank and update the slot with text. This only happens in vue 2, seems to work fine in vue 3.

Comment (ninamarina) [reply packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js:115]: Oh, I took the example from Radio and I see it's not working on Vue 2 there either. Will look into it

Comment (ninamarina) [reply packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js:115]: Seems like it's a problem with SB 7 https://github.com/storybookjs/storybook/issues/16750
It does work when clicking "remount component" though

Comment (braddialpad) [reply packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.stories.js:115]: interesting.. lets not worry about it then.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**